CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- To support future use case that allows attendees with mic issue to at least join the call asap and receive conference audio, update `checkAudioConnectivity` logic inside `DefaultMeetingReadinessChecker` accordingly, to check for audio packets received metrics using `metricsDidReceive` observer in addition to attendee presence condition. Previously the checkAudioConnectivity used to implicitly test that audio was both being sent out and being received but now it's only checking that the audio is being received successfully.
```

Comment:
Couple of edits: 
`To support future use case of allowing users to join a meeting without the need for a microphone or empty audio stream`
And
`Previously, the audio connectivity check used to implicitly test that audio was being sent out from the client and being received by the server. Now this check will include a test for attendee presence, showing that the signal connection has been established, and audio packets being received, showing that the data connection has been established.`

Reply:
Updated. Thanks